src/main/java/net/fabricmc/loader/launch/knot/KnotClient.java

```diff
@@ -22,6 +22,7 @@
 
 public class KnotClient {
 	public static void main(String[] args) {
+		System.setProperty("log4j.configurationFile", "classpath:log4j2.fabric.xml");
```

Comment:
Only when the property is not set, please!

Reply:
Im thinking this might be best set via DLI or something.